src/stories/Filter/components/FilterPage/filterPageUtil.tsx

```diff
@@ -62,6 +63,11 @@ export const FILTERS = {
             parseInitialFilterValue: (browserQueryUrlValue: string): string => browserQueryUrlValue || 'lakefront',
             getFilterBarLabel: (value: string) => `Text Filter: ${value}`
         }
+    ),
```

Comment:
Can you move this up so it's in alphabetical order?

Reply:
Done.
![image](https://user-images.githubusercontent.com/20821671/125652113-3249013a-901b-4fb2-8907-f40555ec4948.png)